glyphs-reader/src/font.rs

```diff
@@ -540,6 +542,15 @@ impl PlistParamsExt for Plist {
             .collect::<Result<_, _>>()
             .ok()
     }
+
+    fn as_gasp_table(&self) -> Option<BTreeMap<i64, i64>> {
+        Some(
+            self.as_dict()?
+                .iter()
+                .filter_map(|(k, v)| Some((k.parse::<i64>().ok()?, v.as_i64()?)))
```

Comment:
An opportunity to use the lovely `Option::zip`, if you wish:
```suggestion
                .filter_map(|(k, v)| k.parse::<i64>().ok().zip(v.as_i64)) 
```

Reply:
TIL